changelog/_unreleased/2020-09-27-improve-REST-API-fetch-from-url.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Change check for Content-Type in fetchfile()
+issue: (issue)[https://github.com/shopware/platform/issues/1372]
+author: Marcel Sotiropoulos
+author_email: marcel.sotiropoulos@soti-it.at
+---
+# API
+*  Changed the check for Content-Type in MediaService.php in function fetchFile(...) to  
```

Comment:
the changelog is now wrong :wink: 

I also would suggest, that it is not necessary to point out the explicit code change.

what do you think about that?
```suggestion
*  Changed the content type check in `\Shopware\Core\Content\Media\MediaService::fetchFile()` to allow additional information
```

Reply:
Yeah, sorry. I was in a hurry yesterday